### ISSUE TITLE
Please reintroduce the right click menu

### ISSUE THREAD

**Issue by Pikaro** [RequestPolicyContinued/requestpolicy#662]
If I'm not mistaken, the option to set rules through the right click menu has been removed in v1.x. I understand that single click navigation is the style of this time, but to me as someone who has disabled all browser bars and whose navigation bar auto-hides, using the new menu is a pain. So I hope that in the final version, there will at least be an option to enable the old menu - I hope there's not too much new code required for this.

**Comment by myrdd** [comment]
> the option to set rules through the right click menu has been removed in v1.x

That's right, but there's already an issue for getting it back, see #353. The 1.0 branch won't be released as stable before it's implemented. Until then, please use [RPContinued 0.5.29](https://addons.mozilla.org/en-US/firefox/addon/requestpolicy-continued/).

> I hope that in the final version, there will at least be an option to enable the old menu

Do you mean by that you want the "old" menu when you click on the toolbar button?
If yes, it could be quite easy to implement, as soon as #353 is fixed.
If no, we can close this issue.

**Comment by pdk56** [comment]
>I hope that in the final version, there will at least be an option to enable the old menu
>Do you mean by that you want the "old" menu when you click on the toolbar button?

I would also like to see the old menu style make a return, or at least have an option in the settings to be able to switch back to it as I personally find it much quicker to use than the new style that has been introduced in request policy continued.

**Comment by myrdd** [comment]
Alright, I've changed the title of this issue.

**Comment by Pikaro** [comment]
I was originally talking about the right-click "old menu" as I've never used the toolbar button (sorry for being so unspecific). But if there were an option to change the toolbar button behavior, that would be even better - giving the user options for customization always is.

**Comment by myrdd** [comment]
I think that in a WebExtension only HTML-style menus can be created. If so, this would be a wontfix.
